ex2/ex2.c

```diff
@@ -8,7 +8,12 @@
 
 int main(void)
 {
-    // Your code here 
+    FILE *file;
+    file = fopen("text.txt", "r");
+    char hold[100];
+     while (fscanf(file,"%s ", hold)==1) 
+        printf("%s ", hold); 
+    fork();
```

Comment:
Great use of `fscanf` here. Try having both the parent and child processes write to the file after the fork. What behavior do you observe?